godtools/App/Features/LessonSwipeTutorial/Presentation/LessonSwipeTutorialView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  LessonSwipeTutorialView.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 4/7/25.
+//  Copyright © 2025 Cru. All rights reserved.
+//
+
+import SwiftUI
+import Lottie
+
+struct LessonSwipeTutorialView: View {
+    
+    private let topGradientColor = Color.getColorWithRGB(red: 12, green: 37, blue: 50, opacity: 1)
+    private let bottomGradientColor = Color.getColorWithRGB(red: 17, green: 49, blue: 66, opacity: 0.75)
+    
+    @ObservedObject private var viewModel: LessonSwipeTutorialViewModel
+    
+    init(viewModel: LessonSwipeTutorialViewModel) {
+        self.viewModel = viewModel
+    }
+    
+    var body: some View {
+        
+        GeometryReader { geometry in
+            ZStack {
+                
+                LinearGradient(
+                    gradient: Gradient(
+                        colors: [topGradientColor, bottomGradientColor]
+                    ),
+                    startPoint: .top,
+                    endPoint: .bottom
+                )
+                
+                VStack {
+                    
+                    Spacer()
+                    
+                    Text(viewModel.interfaceStrings.title)
+                        .font(FontLibrary.sfProDisplayRegular.font(size: 30))
+                        .foregroundStyle(.white)
+                    
+                    let buttonWidth = geometry.size.width * 194/375
+                    let lottieWidth = buttonWidth * 1.5
+                    let lottieHeightRatio: CGFloat = 243/191
+                    
+                    LottieView(animation: .named(getLottieAnimationFileName()))
```

Comment:
I didn't know LottieView existed.  That might be newer in Lottie.  Thinking I will wrap this in AnimatedSwiftUIView and then can remove UIViewRepresentable.  But that can be a separate ticket.

Reply:
Yeah I think it is newer, made it super simple!